corgidrp/data.py

```diff
@@ -353,33 +357,39 @@ def __init__(self, data_or_filepath, pri_hdr=None, ext_hdr=None, err = None, dq
             else:
                 self.dq = np.zeros(self.data.shape, dtype = int)
 
-            if bias is not None:
-                if (np.shape(self.data)[0],) != np.shape(bias):
-                    raise ValueError("The shape of bias is {0} while we are expecting shape {1}".format(bias.shape, self.data.shape))
-                self.bias = bias.astype(np.float32)
-            else:
-                self.bias = np.zeros(self.data.shape[0], dtype = np.float32)
+            #The default hdu extensions
```

Comment:
Just to clarify, this is to remake the hdu_names since the hudlist passed in may not have ERR, DQ?